src/natcap/invest/ui/carbon.py

```diff
@@ -31,13 +31,6 @@ def __init__(self):
             label='Carbon Pools',
             validator=self.validator)
         self.add_input(self.carbon_pools_path)
-        self.cur_lulc_year = inputs.Text(
-            args_key='lulc_cur_year',
-            helptext='The calendar year of the current scenario.',
-            interactive=False,
-            label='Current Landcover Calendar Year',
-            validator=self.validator)
-        self.add_input(self.cur_lulc_year)
         self.calc_sequestration = inputs.Checkbox(
             helptext=(
                 "Check to enable sequestration analysis.  This "
```

Comment:
removing extra space char
```suggestion
                "Check to enable sequestration analysis. This "
```